data/schema_template.yaml

```diff
@@ -0,0 +1,24 @@
+source: the data provider: Eurostat, NOMIS, etc.
+description: a brief description of the data source and transformations that have been applied
+api: the type of API to be used to fetch data: REST, download etc.
+api_documentation: the URL of a documentation page with descriptions of the field meaning (if available)
+endpoint_source: the URL of the api endpoint at which queries or the network requests for dumping the whole dataset can be executed
+enpoint_static: S3 URL of the data file
+framework_id: indicator framework id (from data/framework.json) 
+is_experimental: boolean, whether this indicator is experimental
+year_range: [min year, max year]
+nuts_level: int
+schema:
+  nuts_id:
```

Comment:
(sorry we forgot to discuss this in person) I think this would be just:

```yaml
nuts_id: string
nuts_year: int
...
```

As for `nuts_year` I'd explain in a comment which one of `year_spec` or `year_enforced`